algebird-core/src/main/scala/com/twitter/algebird/MomentsGroup.scala

```diff
@@ -29,14 +29,23 @@ case class Moments(m0: Long, m1: Double, m2: Double, m3: Double, m4: Double) {
   def mean = m1
 
   // Population variance, not sample variance.
-  def variance = m2 / count
+  def variance = if (count > 1)
```

Comment:
what if we do this: add `varianceUnsafe: Double` (and similar) that don't check, and then make `def variance = if (count > 1) varianceUnsafe else Double.NaN`

Is that of any value?